app/libs/file.js

```diff
@@ -40,7 +40,9 @@ function deleteTrack(allTracks, musicFilesFileId) {
     for (let i = 0; i < allTracks.length; i++) {
         let t = allTracks[i];
         if (musicFilesFileId.indexOf(t.get('ressource').fileID) <= -1) {
```

Comment:
You can rely on `_.includes`, which is way more understandable than the Array.indexOf method:

```js
if (!_.includes(musicFilesFileId, t.get('ressource').fileID)) { ... }
```